src/components/Uni/formatsRegistry.ts

```
@@ -33,10 +40,54 @@ export const compactCurrencyFormatter = new Intl.NumberFormat(language, {
   maximumFractionDigits: 0,
 });
 
+function formatFileSize(sizeBytes: number): string {
+  if (sizeBytes > 1000000000) {
+    return `${parseFloat((sizeBytes / 1000000000).toFixed(2))} GB`;
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)